src/stores/AccountController.ts

```diff
@@ -176,34 +187,27 @@ export const logout = async () => {
 
     await restoreFavorites();
     await restoreWatchHistory();
+
+    // it's needed for the InPlayer SDK
     await accountService.logout();
   });
 };
 
 export const register = async (email: string, password: string) => {
-  await useConfig(async ({ cleengId, cleengSandbox }) => {
-    const localesResponse = await cleengAccountService.getLocales(cleengSandbox);
-
-    if (localesResponse.errors.length > 0) throw new Error(localesResponse.errors[0]);
-
-    const responseRegister = await cleengAccountService.register(
-      {
-        email: email,
-        password: password,
-        locale: localesResponse.responseData.locale,
-        country: localesResponse.responseData.country,
-        currency: localesResponse.responseData.currency,
-        publisherId: cleengId,
-      },
-      cleengSandbox,
-    );
-
-    if (responseRegister.errors.length) throw new Error(responseRegister.errors[0]);
-
-    await getAccount(responseRegister.responseData);
```

Comment:
I think there are some pieces missing by removing the getAccount (i.e. the stuff in afterLogin.) Most of it probably doesn't matter for the register flow (there shouldn't be any subscription or shelf data) but the Consents stuff is probably needed.